lib/statsd.js

```diff
@@ -464,5 +462,37 @@ Client.prototype.childClient = function(options) {
   return new ChildClient(this, options);
 };
 
+// Overrides tags in parent with tags from child with the same name (case sensitive) and return the result as new
+// array. parent and child are not mutated.
+function overrideTags (parent, child) {
+  var childCopy = {};
+  var toAppend = [];
+  child.forEach(function (tag) {
+    var idx = tag.indexOf(':');
```

Comment:
You can (accidentally) add null or undefined to an array, so there should be a check here for this